<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>83</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PromptDialog demo</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Click on any of the buttons to create a PromptDialog.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="promptBtn">
        <property name="text">
         <string>Prompt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="promptBtn2">
        <property name="text">
         <string>Prompt 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="staticPromptBtn">
        <property name="text">
         <string>Static prompt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
